### PR TITLE
Gp2-1632-generic-ep-pages

### PR DESCRIPTION
**Pull request by depsiatwal**
(Please add a sentence or two explaining the context of this PR. Reviewers can go to the ticket for detail.)
CONTEXT: This changeset adds/removes/updates [summary of feature/area/code] so that [summary of benefit].

_Tick or delete as appropriate:_

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by depsiatwal** [comment]
This PR is WIP it can be merged so that users can see progress on PDF download in dev

**Comment by dahfool** [comment]
@jakehopking  If you want to reuse the sass variables like colors, add a core/sass/pdf_download.scss and a script to package.json which will compile to exportplan/templates/exportplan/pdf_download.css

File: exportplan/core/parsers.py [hunk 1]
```diff
@@ -11,15 +13,78 @@ class ExportPlanParser:
...
     def __init__(self, data):
         self.data = data
         self.seralizer = serializers.ExportPlanSerializer(data=self.data)
         self.seralizer.is_valid()
+        self.serialize_for_template()
 
     def __bool__(self):
         return bool(self.data)
 
+    def get_key(self, key):
```
Comment (bobmeredith) [inline exportplan/core/parsers.py:30]: get_key and set_key look really generic. Should they go in core utils perhaps.

**Review by bobmeredith — APPROVED** [review]
LGTM - with a suggestion

**Pull request merged**